test/INP_concentration_parametrization.jl

```diff
@@ -0,0 +1,34 @@
+import Test as TT
+
```

Comment:
Maybe we can move it into the heterogeneous ice nucleation tests? It will become an option for parameterizing immersion freezing in the end